### DISCUSSION TITLE
错误代码: 0xC004F017

### DISCUSSION THREAD

**Discussion by Tobias-Lan** [YerongAI/Office-Tool#700] [Q&A]
Office 16, Office16O365HomePremR_Grace edition
•	描述: Office 16, RETAIL(Grace) channel
•	错误代码: 0xC004F017

**Comment by iDolmatov** [comment]
Grace is similar to trial, but without the activation function. It is logical that there will be an error.

How to check error description:
![Office_Tool_Plus_2U4BI1tdyS](https://github.com/YerongAI/Office-Tool/assets/19782385/0cdbd29c-f2e1-4a3c-8fae-13679e62fd5e)

**Reply by kaze0617** [reply]
You better use kms to active your Office.

**Comment by kaze0617** [comment]
You better use kms to active your Office.
